server/middleware/validations/profile.js

```diff
@@ -0,0 +1,92 @@
+const { check, validationResult } = require('express-validator/check');
+
+const profileValidator = {
+  newProfileValidator: [
+    check('age')
+      .not().isEmpty()
+      .withMessage('Age is required')
+      .trim()
+      .isNumeric()
+      .withMessage('Enter a valid age'),
+    check('gender')
+      .not().isEmpty()
+      .withMessage('Gender is required')
+      .trim()
+      .isIn(['male', 'Male', 'female', 'Female'])
```

Comment:
Consider a Bobrisky situation here for inclusion and diversity?

Reply:
lol okay. I would take down this validation.